languages/go/exercises/concept/basic-strings/.docs/after.md

```diff
@@ -0,0 +1,12 @@
+# After
+
+### Slice indexes
+
+When addressing an index or index range in a slice make sure it exists.
+
+Code that does not check if an index exists in a `slice`, will `panic` if that index does not exist.
+A `panic` should be considered a bug in the code to be fixed. There will be more on panics later.
```

Comment:
Perhaps add a quick example that reminds users slices are `0-indexed`?

```
Example: If we have a slice that is 4 elements long, valid index values would be 0, 1, 2, and 3. If we tried to access anything greater, like mySlice[4] this would cause a [panic](https://play.golang.org/p/2npb6GKtEMD).

panic: runtime error: index out of range [4] with length 4

goroutine 1 [running]:
main.main()
	/tmp/sandbox933015469/prog.go:21 +0x1a0
```

Reply:
Could be helpful, yes. Although the slices exercise is a prerequisite and there I'm testing very thoroughly for this. So I was already thinking of removing this.

Also note that this is in the `after.md`. So the student sees this after successfully solving the exercise. Not sure if it is helpful at all (I mean my comments in line 7+8, not your suggestion).

What do you think?

Reply:
Ah, that's fair. With that in mind I would move the two likes 7-8 out to the hints section (and out from `after.md`).

Reply:
Right, good idea. Will do that later.